migrations/sql-migrations/migrations/20220304184711_schema.js

```diff
@@ -0,0 +1,165 @@
+
+const setOnUpdateTrigger = (table) => (`
```

Comment:
this one should be deleted as we have it duplicated in the API right? the one in the API will be auto-released into `migrations/sql/production|staging`